random-menu.rb

```diff
@@ -0,0 +1,69 @@
+# User input on number of items that will be generated
+print "\n----- Welcome to RandPal's American Stackhouse! -----
+\r☆ Hope you're hungry as hippos, friend ☆ \n"
+puts "\nHow many items do you want to eat? (Up to 10)"
+create = gets.chomp.to_i
+
+# Random numbered arrays for each food-related array
+random_one = []
+until random_one.length == create
+  num1 = rand(create)
+  random_one << num1 unless random_one.include? num1
```

Comment:
I like that instead of shuffling the array itself, you created a randomized list of indices. Very clever.